src/dc_imex.rs

```diff
@@ -805,115 +749,92 @@ unsafe fn export_backup(context: &Context, dir: *const libc::c_char) -> bool {
 /*******************************************************************************
  * Classic key import
  ******************************************************************************/
-unsafe fn import_self_keys(context: &Context, dir_name: *const libc::c_char) -> libc::c_int {
+fn import_self_keys(context: &Context, dir_name: &str) -> bool {
```

Comment:
would be nice if this returned a `Result` as well

Reply:
does now